### PR TITLE
linux: Foundational_Components_IPC_J7*: Fix link to Top Level Makefile

### PR DESCRIPTION
**Pull request by aniket-l**
Signed-off-by: Aniket Limaye <a-limaye@ti.com>

### CODE REVIEW

File: source/linux/Foundational_Components_IPC_J7200.rst
Comment (nmenon) [inline source/linux/Foundational_Components_IPC_J7200.rst:1]: Please always add a commit message

Comment (StaticRocket) [reply source/linux/Foundational_Components_IPC_J7200.rst:1]: Is this really necessary? I've see some people will leave out the body of the message if it ends up being almost identical to the subject.

**Review by StaticRocket — APPROVED** [review]
I approve of the content changes. I'll let @nmenon argue guidelines.

File: source/linux/Foundational_Components_IPC_J7200.rst
Comment (nmenon) [reply source/linux/Foundational_Components_IPC_J7200.rst:1]: At least document a procedure and follow something. Kernel folks like our commit messages.

Comment (aniket-l) [reply source/linux/Foundational_Components_IPC_J7200.rst:1]: Have added some commit msgs
But also did we reallly decide on some guidelines/procedure?

**Comment by nmenon** [comment]
rebase on master to get the checkers to run

**Comment by aniket-l** [comment]
> rebase on master to get the checkers to run
@nmenon @StaticRocket 
Not sure what I am missing here... I seem to have added necessary formatting to the commits
```
Type commit_signoff check failed => 04a924b2eb59a5baf41d57c4484dc3c4ef52756c 
It doesn't match regex: Signed-off-by:.*[A-Za-z0-9]\s+<[\w\.]+@([\w-]+\.)+[\w-]{2,4}>
```
Looks to me like below change is needed in the regex matching pattern, to match with the '-' character in email-id?
`<[\w\.]+@([\w-]+\.)+[\w-]{2,4}>` => `<[\w\.\-]+@([\w-]+\.)+[\w-]{2,4}>`

**Comment by StaticRocket** [comment]
Yeah, that's a bug in the upstream checker. They should not attempt to verify email addresses with regex. The presence of non-whitespace characters around an @ should be more than enough.

**Review by nmenon — APPROVED** [review]

**Comment by StaticRocket** [comment]
@cshilwant @praneethbajjuri we need another maintainer to sign off on this.

**Review by cshilwant — APPROVED** [review]

**Review by praneethbajjuri — APPROVED** [review]

**Pull request merged**